assets/src/components/common/ActivityXmlEditor.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import AceEditor from 'react-ace';
+import 'ace-builds/src-noconflict/mode-javascript';
+import 'ace-builds/src-noconflict/theme-xcode';
+
+export type ActivityXmlEditorProps = {
+  value: string;
+  onChange: (newValue: string) => void;
+  disabled: boolean;
+};
+
+export const ActivityXmlEditor = (props: ActivityXmlEditorProps) => {
```

Comment:
maybe should just name `XmlEditor` ? this isn't specific to activities

Reply:
Name refactored